pkg/azurefile/controllerserver.go

```diff
@@ -797,7 +807,18 @@ func (d *Driver) CreateSnapshot(ctx context.Context, req *csi.CreateSnapshotRequ
 		return nil, status.Error(codes.InvalidArgument, "CreateSnapshot Source Volume ID must be provided")
 	}
 
-	mc := metrics.NewMetricContext(azureFileCSIDriverName, "controller_create_snapshot", d.cloud.ResourceGroup, d.cloud.SubscriptionID, d.Name)
+	rgName, _, _, _, _, subsID, err := GetFileShareInfo(sourceVolumeID)
```

Comment:
maybe it's better to return a struct pointer here since there is already golint error here: `declaration has 4 blank identifiers`

Reply:
Could I workaround it by adding comment to exclude these `dogsled` linters this time? Since too many places to fix which raises the risk of making mistake and makes the code review more difficult, while the improvement to the code is very limited.